src/test/resources/test-postgres-rutherford-create-script.sql

```diff
@@ -0,0 +1,1232 @@
+-- SCHEMA
```

Comment:
This seems like a recipe for disaster, or at least a doubling of the maintenance burden of maintaining the database schema? Surely this could just be a data-only dump, and then you just load the three usual init scripts and then this one?
Although [the code looks like it's script singular](https://github.com/testcontainers/testcontainers-java/blob/e7cdfa31af08074aa9dbb3f3d21a622144995b9d/modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java#L136-L139=), which is very annoying!

Reply:
Yes, the DB dump could be vastly improved. Again, priorities :) Not sure what you mean by script singular?

Reply:
I've been stripping down my local DB regularly and using it to generate this dump. It's a less than ideal solution but it's quick and easy enough to add/change data while working on tests. Perhaps keeping a separate DB and switching between the two would be even better to enable regular work alongside test work when we'll be in a more mature state. The downside will be having to run any migration twice.

There is one advantage to running a full dump like this as opposed to concatenating different parts: the dump will always be in a coherent state.